doc/api/axis_api.rst

```diff
@@ -1,12 +1,954 @@
-****
-axis
-****
+===================
+ axis and tick API
+===================
 
-
-:mod:`matplotlib.axis`
-======================
+.. contents:: Table of Contents
+   :depth: 3
+   :local:
+   :backlinks: entry
 
 .. automodule:: matplotlib.axis
-   :members:
-   :undoc-members:
-   :show-inheritance:
+
+Inheritance
+===========
+
+.. inheritance-diagram:: Tick Ticker XAxis YAxis XTick YTick
+   :private-bases:
```

Comment:
And I now see this is really at the top so I am not sure what I was remebering...